api/src/main/resources/liquibase-update-to-2.0.xml

```diff
@@ -8774,7 +8774,7 @@
             </column>
             <column name="allergen_type" type="varchar(50)">
             <constraints nullable="false"/></column>
-            <column name="comment" type="varchar(1024)"></column>
```

Comment:
Can you add a new changeset for renaming the column instead of modifying an existing one?

Reply:
Okay